.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
Add an empty line at the end of your code

Reply:
More like this shouldnt be here at all